config/pepperConfig.js.ctmpl

```diff
@@ -95,6 +95,22 @@ var DDP_ENV = {
     {{end}}
 {{end}}
 
+{{if eq $study_guid "cmi-brain-v2"}}
```

Comment:
New study guid.

Reply:
Creating new app on frontend is a good idea. Creating new study on backend will be bad... A lot of backend configuration, users, and data is tied to the `cmi-brain` study. Moving it to a new study `cimi-brain-v2` is like doing a whole study migration...

Reply:
Hmm, can we create custom activities in the current study scope and return them only for specific test users?

Reply:
Might be a bit tricky but certainly do-able.